jena-jdbc/pom.xml

```diff
@@ -45,22 +45,16 @@
 		<module>jena-jdbc-driver-bundle</module>
 	</modules>
 
-  <dependencyManagement>
-    <dependencies>
-      <!-- Log4j2 -->
-      <dependency>
-        <groupId>org.apache.logging.log4j</groupId>
-        <artifactId>log4j-slf4j-impl</artifactId>
-        <version>${ver.log4j2}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.logging.log4j</groupId>
-        <artifactId>log4j</artifactId>
-        <version>${ver.log4j2}</version>
-        <type>pom</type>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
+	<dependencyManagement>
+		<dependencies>
+			<!-- Log4j2 -->
```

Comment:
tabs! (not a blocker but ...)

The tabs vs spaces thing has become less neutral since we all started looking at code via the github and other web interfaces!

Reply:
Yeah these `pom.xml` files were already using tabs so I just reformatted to match existing style

Reply:
But I'm happy to fully reformat these `pom.xml` files to use spaces (I'd prefer that anyway, was just trying to minimise noise in the PR)